go/service/user.go

```diff
@@ -126,13 +129,22 @@ func (h *UserHandler) LoadUserByName(_ context.Context, arg keybase1.LoadUserByN
 	return
 }
 
-func (h *UserHandler) LoadUserPlusKeysV2(netCtx context.Context, arg keybase1.LoadUserPlusKeysV2Arg) (ret keybase1.UserPlusKeysV2AllIncarnations, err error) {
-	netCtx = libkb.WithLogTag(netCtx, "LUPK2")
-	defer h.G().CTrace(netCtx, fmt.Sprintf("UserHandler#LoadUserPlusKeysV2(%+v)", arg), func() error { return err })()
-	p, err := h.G().GetUPAKLoader().LoadV2WithKID(netCtx, arg.Uid, arg.PollForKID)
-	if p != nil {
-		ret = *p
+func (h *UserHandler) LoadUserPlusKeysV2(ctx context.Context, arg keybase1.LoadUserPlusKeysV2Arg) (ret keybase1.UserPlusKeysV2AllIncarnations, err error) {
+	mctx := libkb.NewMetaContext(ctx, h.G()).WithLogTag("LUPK2")
+	defer mctx.Trace(fmt.Sprintf("UserHandler#LoadUserPlusKeysV2(%+v)", arg), func() error { return err })()
+
+	var retp *keybase1.UserPlusKeysV2AllIncarnations
```

Comment:
maybe `retp := &ret` and then....

Reply:
I think I just didn't want to do a copy. Made the changes.